core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -49,16 +64,21 @@ static PageStore create() {
    * @param dst destination channel to read this new page
    * @return the number of bytes read
    * @throws IOException
+   * @throws PageNotFoundException when the page isn't found in the store
    */
-  int get(long fileId, long pageIndex, WritableByteChannel dst) throws IOException;
+  int get(long fileId, long pageIndex, WritableByteChannel dst) throws IOException,
+      PageNotFoundException;
```

Comment:
instead of a new `PageNotFoundException`, can we reuse `NotFoundException` in alluxio state exception? We tend to stop introducing specific Alluxio exceptions but standardize on AlluxioStateException